pkgs/development/python-modules/djangorestframework-stubs/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, django-stubs
+, fetchFromGitHub
+, mypy
+, py
+, pytest-mypy-plugins
+, pytestCheckHook
+, pythonOlder
+, requests
+, types-pyyaml
+, types-requests
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "djangorestframework-stubs";
+  version = "3.14.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "typeddjango";
+    repo = "djangorestframework-stubs";
+    rev = version;
+    hash = "sha256-T357ocJvDC+vt0I4VyAu0Q9YzY9cSK7shgp9fQ1qHyY=";
+  };
+
+  propagatedBuildInputs = [
+    django-stubs
+    mypy
+    requests
+    types-pyyaml
+    types-requests
+    typing-extensions
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mypy
+  ];
+
+  checkInputs = [
+    py
+    pytest-mypy-plugins
```

Comment:
These inputs should go to `nativeCheckInputs`.

Reply:
Thanks, updated!